src/universal/conf/app.conf

```diff
@@ -0,0 +1,23 @@
+# This is the base configuration file for the Mantis ETC client.
+
+# This where all the default settings are defined (this file is packaged within the executable).
+# It should always go at the top.
+include "application.conf"
+
+# Enable HTTPS
```

Comment:
This was a leftover copy from network.conf, does it still make sense? Shouldn't it be enabled in the application.conf?